product_docs/docs/pgd/5/terminology.mdx

```diff
@@ -2,83 +2,116 @@
 
 #### Failover
 
-The automated process that recognizes a failure in a highly available database cluster and takes action to connect the application to another active database. The goal is to minimize downtime and data loss.
+The automated process that recognizes a failure in a highly available database cluster and takes action to maintain consistency and availability. The goal is to minimize downtime and data loss.
 
-#### Horizontal scaling or scale out
+#### Group commit
 
-A modern distributed computing approach that manages workloads across multiple nodes, such as scaling out a web server to handle increased traffic.
+A synchronous commit mode which requires more than one PGD node to successfully receive and confirm a transaction at commit time.
 
-#### Logical replication
+#### Immediate consistency
 
-Provides more flexibility than physical replication in terms of selecting the data replicated between databases in a cluster. Also important is that cluster members can be on different versions of the database software.
+A distributed computing model where all replicas are updated synchronously and simultaneously. This ensures that all reads after a write has completed will see the same value on all nodes. The downside of this approach is its negative impact on performance.
 
-#### Nines
+#### Logical replication
 
-A measure of availability expressed as a percentage of uptime in a given year. Three nines (99.9%) allows for 43.83 minutes of downtime per month. Four nines (99.99%) allows for 4.38 minutes of downtime per month. Five nines (99.999%) allows for 26.3 seconds of downtime per month.
+A more efficient method of replicating changes in the database. While physical streaming replication duplicate the originating database’s disk blocks, logical replication instead sees the DML commands - inserts, deletes and updates,- published to all systems that have subscribed to see the changes. Each subscriber then applies the changes locally. Logical replication is not able to support most DDL
 
 #### Node
 
-One database server in a cluster. A term *node* differs from the term *database server* because there's more than one node in a cluster. A node includes the database server, the OS, and the physical hardware, which is always separate from other nodes in a high-availability context.
+A general term for an element of a distributed system. A node can play host to any service. In PGD, there are [PGD Nodes](#pgd-node) which run a Postgres database and the BDR extension and optionally a PGD Proxy service.
+
+Typically, for high availability, each node runs on separate physical hardware, but not necessarily. For example, in modern cloud platforms the hardware may be shared with the cloud.
```

Comment:
Would a proxy being colocated w/ the PGD node be a better example? The cloud is obviously cloudy, anyway.

Reply:
```suggestion
Typically, for high availability, each node runs on separate physical hardware, but not necessarily. For example, a proxy might share a hardware node with a database.
```